acceptance/orgs_test.go

```diff
@@ -0,0 +1,36 @@
+package acceptance_test
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Orgs", func() {
+	It("has a default org", func() {
+		orgs, err := Carrier("orgs", "")
+		Expect(err).ToNot(HaveOccurred())
+		Expect(orgs).To(MatchRegexp("workspace"))
+	})
+
+	Describe("create-org", func() {
+		It("creates and targets an org", func() {
+			By("creating an org")
+			_, err := Carrier("create-org mycreatedorg", "")
+			Expect(err).ToNot(HaveOccurred())
+			orgs, err := Carrier("orgs", "")
+			Expect(err).ToNot(HaveOccurred())
+			Expect(orgs).To(MatchRegexp("mycreatedorg"))
+
+			By("targeting an org")
+			_, err = Carrier("target mycreatedorg", "")
+			Expect(err).ToNot(HaveOccurred())
+			out, err := Carrier("target", "")
+			Expect(err).ToNot(HaveOccurred())
+			Expect(out).To(MatchRegexp("Currently targeted organization: mycreatedorg"))
+
+			By("switching org back to default")
+			_, err = Carrier("target workspace", "")
+			Expect(err).ToNot(HaveOccurred())
+		})
+	})
+})
```

Comment:
Future: Todo: `listing orgs`

Reply:
Same as above, we use the `carrier orgs` command to verify org creation and targeting.

Reply:
Ditto. Missed, and is visually part of the create block.